apps/simulator/src/components/Toolbar/Toolbar.tsx

```diff
@@ -30,23 +31,30 @@ export const ToolbarSettings: React.FC = () => {
     },
   };
 
+  const toolTipData = [
+    {icon: <ZoomOutIcon fontSize="medium" onClick={handleZoomOutText}/>, title: "Zoom Out"},
+    {icon: <RestartAltIcon fontSize="medium" onClick={handleResetTextZoom}/>, title: "Reset"},
+    {icon: <ZoomInIcon fontSize="medium" onClick={handleZoomInText}/>, title: "Zoom In"}
+  ]
+
   return (
     <div>
-      <Button sx={buttonStyle} aria-describedby={id} type="button" onClick={handleClick}>
-        <SettingsIcon fontSize="large" sx={buttonStyle.IconStyle} />
-      </Button>
+      <CustomTooltip title="Settings">
+        <Button sx={buttonStyle} aria-describedby={id} type="button" onClick={handleClick}>
+          <SettingsIcon fontSize="large" sx={buttonStyle.IconStyle} />
+        </Button>
+      </CustomTooltip>
       <Popper id={id} open={open} anchorEl={anchorEl}>
         <Box sx={{ border: 1, p: 1, bgcolor: '#007AB7' }}>
           <div className="flex items-center justify-center">
-            <Button sx={iconStyle} onClick={handleZoomOutText}>
-              <ZoomOutIcon fontSize="medium" />
-            </Button>
-            <Button sx={iconStyle} onClick={handleResetTextZoom}>
-              <RestartAltIcon fontSize="medium" />
-            </Button>
-            <Button sx={iconStyle} onClick={handleZoomInText}>
-              <ZoomInIcon fontSize="medium" />
-            </Button>
+            {toolTipData.map((item,index)=>(
+              <CustomTooltip title={item.title}>
+                <Button sx={iconStyle}>
+                  {item.icon} 
+                </Button>
+            </CustomTooltip>
+            ))}
```

Comment:
You didn't used the index, I believed there will be a warning on console when you didn't put a unique "key" prop in a list, 
```suggestion
            {toolTipData.map((item,index)=>(
              <CustomTooltip key={index} title={item.title}>
                <Button sx={iconStyle}>
                  {item.icon} 
                </Button>
            </CustomTooltip>
            ))}
```